spec/cobrato/resources/regress_cnab_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'spec_helper'
+
+describe Cobrato::Resources::RegressCnab do
+  let(:http)          { Cobrato::Http.new('45d4e96c707f2a45f73ac9848ff8eeab') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.